config/application.rb

```diff
@@ -23,5 +23,7 @@ class Application < Rails::Application
     #
     # config.time_zone = "Central Time (US & Canada)"
     # config.eager_load_paths << Rails.root.join("extras")
+
+    config.autoload_paths += %W[#{config.root}/app/presenters]
```

Comment:
Based on our discussion, presenters are a pretty standard pattern for Rentable and adding this here is pro-forma and just for the purposes of our exercise (otherwise, I'd probably want to have a discussion as a team about whether Presenters are solving the problem we want them to - though I think the fact we are using them currently is a strong "yes" to that). :)